src/node.cc

```diff
@@ -4092,7 +4087,13 @@ inline void PlatformInit() {
       ABORT();
   }
 
+#if HAVE_INSPECTOR
+  sigset_t sigmask;
+  sigemptyset(&sigmask);
+  sigaddset(&sigmask, SIGUSR1);
+  const int err = pthread_sigmask(SIG_SETMASK, &sigmask, nullptr);
   CHECK_EQ(err, 0);
+#endif  // HAVE_INSPECTOR
```

Comment:
I don't want to make this a torment for you but the reason SIGUSR1 is blocked first thing is to make the 'kill window' as small as possible.  Some tools start node and immediately send a SIGUSR1 - if the signal isn't masked, it kills the process.